io/plugins/eu.esdihumboldt.hale.io.jdbc.spatialite/src/eu/esdihumboldt/hale/io/jdbc/spatialite/internal/SpatiaLiteHelper.java

```diff
@@ -106,8 +106,8 @@ public Boolean call() throws Exception {
 			if (!result) {
 				String msg = "The SpatiaLite extension for SQLite (mod_spatialite) could not be loaded. "
 						+ "This means that it is not possible to properly load or store geometries from/to SQLite.\n\n"
-						+ "Please check the hale studio help on how to make it available:\n"
-						+ "hale studio User Guide > Reference > Supported formats > Import > SQLite and SpatiaLite";
+						+ "Please check the hale«studio help on how to make it available:\n"
```

Comment:
I believe that by mistake the reverse operator was inserted `hale«studio`

Reply:
> just in one place the operator is reversed.

Good catch, thanks!

> Should these changes also be a part of 5.0.0 release?

I'd say no. Even though it's a cosmetic change, I think it needs to be thoroughly tested on all platforms. Also, the help still needs to be updated and I'd say that both should be released together.